src/apps/api/mixins.py

```diff
@@ -0,0 +1,18 @@
+class DefaultUserCreateMixin:
+    """This will automatically set `YourModel.user` to `request.user`. To override which
+    attribute the user is written to, add a `user_field` to your classes Meta information
+
+    Example:
+        class MySerializer(DefaultUserCreateMixin, ModelSerializer):
+            class Meta:
+                model = YourModel
+                # YourModel.owner = a foreign key to request.user
+                user_field = 'owner'
+    """
+    def create(self, validated_data):
+        user_field = getattr(self.Meta, 'user_field', 'user')
```

Comment:
should this default to `created_by` instead? since that is the convention for most models in this codebase?

Reply:
_maybe?_ I'd almost like to make a `django-speedy-boys` (or some other funny name) repo where we can keep these mixins centralized... 

the default feels pretty arbitrary, could be `user` or `created_by`.. or `owner`, could make the default set in settings? `settings.DRF_DEFAULT_CREATE_USER_FIELD_NAME`